backend/cmm_helpers.mli

```diff
@@ -381,28 +380,18 @@ val bigarray_elt_size_in_bytes : Lambda.bigarray_kind -> int
     bigarray. *)
 val bigarray_word_kind : Lambda.bigarray_kind -> memory_chunk
 
-(** Operations on 32-bit integers *)
-
-(** [low_32 _ x] is a value which agrees with x on at least the low 32 bits *)
-val low_32 : Debuginfo.t -> expression -> expression
+(** Operations on n-bit integers *)
 
-(** Sign extend from 32 bits to the word size *)
-val sign_extend_32 : Debuginfo.t -> expression -> expression
+(** Simplify the given expression knowing low [bits] bits will be irrelevant *)
+val ignore_low_bits : bits:int -> dbg:Debuginfo.t -> expression -> expression
 
-(** Zero extend from 32 bits to the word size *)
-val zero_extend_32 : Debuginfo.t -> expression -> expression
+(** Simplify the given expression knowing that bits other than the low [bits] bits will be
+    irrelevant *)
+val low_bits : bits:int -> dbg:Debuginfo.t -> expression -> expression
 
-(** Operations on 63-bit integers. These may only be used for compilation to
-    64-bit targets. *)
+val sign_extend : bits:int -> dbg:Debuginfo.t -> expression -> expression
```

Comment:
missing doc comment here and for `zero_extend` (especially considering that the "old" functions had doc comments)